Assets/HoloToolkit/Input/README.md

```diff
@@ -189,16 +190,30 @@ or in your Visual Studio Package.appxmanifest capabilities.
 **RecognizerStart** Set this to determine whether the keyword recognizer will start immediately or if it should wait for your code to tell it to start.
 
 #### Voice
-##### KeywordManager.cs
-Allows you to specify keywords and methods in the Unity Inspector, instead of registering them explicitly in code.  
-**IMPORTANT**: Please make sure to add the microphone capability in your app, in Unity under  
+
+**IMPORTANT**: Please make sure to add the Microphone and Music Library capabilities in your app, in Unity under  
```

Comment:
I copied this portion from MicStream.cs section above and didn't notice the slight difference. I placed this just below the Voice heading as it applies to all the section. I thought it wouldn't make sense to duplicate the warning for each voice handler.